app/models/exchange_rate_country_currency.rb

```diff
@@ -13,4 +13,29 @@ def validate
     validates_presence :currency_code
     validates_presence :validity_start_date
   end
+
+  def self.live_currency_codes
+    # Criteria:
+    # End date within the current month
+    # Start date less than or equal to the end of the current month
+
+    where(
+      (Sequel[:validity_end_date] =~ (Time.zone.today.beginning_of_month..Time.zone.today.end_of_month)) |
+      (Sequel[:validity_end_date] =~ nil) &
+      (Sequel[:validity_start_date] <= Time.zone.today.end_of_month),
+    ).distinct(:currency_code).select_map(:currency_code)
+  end
+
+  def self.live_last_twelve_months(date)
+    # Criteria:
+    # Start date less than then end of the current month and end date nil or more than
+    # End date more than then beginning of the current month 12 months ago
```

Comment:
Maybe we could make this phrase more readable. or, if we can't,  even remove it since it does not add much to the code?